src/cpr_sdk/parser_models.py

```diff
@@ -412,28 +426,31 @@ def to_passage_level_json(self, include_empty: bool = True) -> list[dict[str, An
             )
         )
 
-        empty_html_text_block_keys: list[str] = list(HTMLTextBlock.model_fields.keys())
-        empty_pdf_text_block_keys: list[str] = list(PDFTextBlock.model_fields.keys())
+        empty_html_text_block_keys: list[str] = self._rename_text_block_keys(list(HTMLTextBlock.model_fields.keys()))  # type: ignore
```

Comment:
Just looking at the `# type: ignore` I'm assuming this is as we may be returning a dict from the static method? Could be good to get a test in for this functionality.

Reply:
imo because it's just a dict and a list comprehension it doesn't really need a test? this is just pyright being weird. but happy to write one if you think it's worth it